_visualizations/pie-chart.md

```diff
@@ -17,7 +17,7 @@ components:
 lead: A pie chart is a circular graph that presents values as proportionate slices.
 tags: [comparative]
 guidance-always:
-- Always consider using a bar chart or displaying values directly before using a pie chart.
+- Always consider using a [bar chart](!BASEURL!/visualizations/bar-chart) or displaying values directly before using a pie chart.
```

Comment:
@lukad03 Just added the one example link you mentioned - didn't go through and look for where others should be.

Reply:
@afeld this is great! I will pull this branch down and add the other `bar chart` example directly below this and then it should be ready for merging.

I'll follow up with you about some broader questions I have about PRs, merging policy, etc